Quadriga/src/main/webapp/WEB-INF/views/workspace/workspacedetails.jsp

```diff
@@ -467,8 +473,12 @@ cursor: pointer;
                                 changedTextFileAccess = "Public";
                             }
                             
-                            var content = "Are you sure you want to make this text file " + changedTextFileAccess + "?";
```

Comment:
Regarding the if-statement before: simply create a map {textFileAccess -> string that should be displayed}. This way you can simply get the string you need and  it's easily extendable.

(I don't seem to have looked too closely at this file before..., sorry).